indico/modules/events/templates/reviews/rating_widget.html

```diff
@@ -8,12 +8,12 @@
             </span>
         </div>
         <div class="question-text f-self-stretch">
-            {{ field.label.text }}
-            {%- if field.flags.required %}
+            {{- field.label.text -}}
+            {%- if field.flags.required -%}
```

Comment:
Why truncate whitespace here? IMO it looks quite weird with no space between the question and the asterisk